README.md

```diff
@@ -6,6 +6,78 @@
 
 Django-components is a package that introduces component-based architecture to Django's server-side rendering. It aims to combine Django's templating system with the modularity seen in modern frontend frameworks.
 
+A component in django-components can be as simple as a Django template and Python code to declare the component:
```

Comment:
Recently when I shared an update on Reddit about new feature in django-components, there was one guy who thought, based on the link I shared, that django-components supports only the "inlined" HTML/CSS/JS. What probably led him to it is that for most of the documentation I wrote, I used the inlined format.

So I think it would be good to show both formats right at the start. So I added this section both here to the root README, and to the documentation welcome page (`welcome.md`).